components/cards/LazyListBookCard.vue

```diff
@@ -99,9 +101,21 @@ export default {
     mediaType() {
       return this._libraryItem.mediaType
     },
+    duration() {
+      return this.media.duration || null
+    },
     isPodcast() {
       return this.mediaType === 'podcast'
     },
+    episodes() {
```

Comment:
Computed property should always return some value

Reply:
Whoops! I'll keep that in mind for the future--this is the first time I've used Vue so still learning the ropes.